editor/project_converter_3_to_4.cpp

```diff
@@ -1607,7 +1607,7 @@ int ProjectConverter3To4::convert() {
 	{
 		String conventer_text = "; Project was converted by built-in tool to Godot 4.0";
 
-		ERR_FAIL_COND_V_MSG(!FileAccess::exists("project.godot"), ERROR_CODE, "Current directory doesn't contains any Godot 3 project");
+		ERR_FAIL_COND_V_MSG(!FileAccess::exists("project.godot"), ERROR_CODE, "Current directory doesn't contain any Godot 3 projects");
```

Comment:
```suggestion
		ERR_FAIL_COND_V_MSG(!FileAccess::exists("project.godot"), ERROR_CODE, "Current directory doesn't contain any Godot 3 project.");
```
It can only convert one project at a time, so it shouldn't be plural (and missing dot at end of sentence).

Please amend the commit with `git commit --amend` and update the PR with `git push --force` (see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request)).